client/src/components/previews/LocationPreview.js

```diff
@@ -54,14 +58,37 @@ const LocationPreview = ({ className, uuid }) => {
         <PreviewField label="Name" value={location.name} />
 
         <PreviewField
-          label="Status"
-          value={Location.humanNameOfStatus(location.status)}
+          label="Type"
```

Comment:
It makes sense to keep the order of the Preview and the Show page mostly the same.